frontend/src/general/Header.css

```diff
@@ -1,11 +1,23 @@
-header{
-    top:0;margin-top: 50px;
+header div{
+    display: flex;
+    margin: 30px 0 40px 25px;
 }
 
 h1{
-    font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-family: Trebuchet MS, Helvetica, Arial, Verdana, Tahoma, sans-serif;
     color: #1d721d;
-    margin-bottom: 50px;
+    margin-left: 20px;
+    display: inline-block;
+}
+
+h1 span{
+    font-style: oblique;
+    font-family: bradley hand, Helvetica, Arial, Verdana, Tahoma, sans-serif;
+    font-weight: bolder;
+}
+
+.nav:hover{
+    color: #47cb47;
 }
```

Comment:
Mhhh.... ich finde die Klassen-Namen verwirrend:
.menu ist Klasse für Navigation, .nav ist Klasse für Link in der Navigation...
Klarer wäre .nav für die Navigation und .nav-link für Link in Navigation. Es gibt da auch so Best-Practice für - google doch ganz kurz mal nach BEM-Regeln für Klassen-Namen?